src/Sdk/WebApi/WebApi/HttpClients/HealthHttpClient.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHub.Services.Common;
+using GitHub.Services.WebApi;       
+
+namespace GitHub.Services.Health.Client
+{     
+    public class HealthHttpClient : VssHttpClientBase
```

Comment:
Is this something you made up or it's copied from ADN?
`/src/sdk/*` are codes we copied from ADN.

Reply:
It's actually code from ADO! ADN removed that client code interestingly.

Reply:
We shouldn't add anything to the SDK folder if the code is not sourced in ADN.

Reply:
Removed the class entirely, and calling the health endpoint directly